webview/src/setup/components/App.test.tsx

```diff
@@ -187,6 +186,28 @@ describe('App', () => {
       })
     })
 
+    it('should show "Set Env" and "Install (pip)" button tooltips when dvc is unavailable and Python extension is not installed', () => {
```

Comment:
My original plan was to add tests for checking if buttons and tooltips were disabled depending on circumstances but I for some reason, tooltips and button weren't disabling in the tests. They are working in Storybook and in development but not in testing. Any ideas what's going on?

Reply:
the magic virtual dom probably doesn't set the properly correctly